apps/prairielearn/src/models/enrollment.test.ts

```diff
@@ -83,12 +83,13 @@ describe('ensureEnrollment', () => {
     });
 
     await queryRow(
-      `INSERT INTO enrollments (user_id, course_instance_id, status)
-       VALUES ($user_id, $course_instance_id, 'blocked')
+      `INSERT INTO enrollments (user_id, course_instance_id, status, first_joined_at)
+       VALUES ($user_id, $course_instance_id, 'blocked', $first_joined_at)
        RETURNING *`,
       {
         user_id: user.user_id,
         course_instance_id: '1',
+        first_joined_at: new Date().toISOString(),
```

Comment:
IIRC `toISOString()` isn't needed here, `pg` should take care of that for us.

Reply:
Done in [d87e16f](https://github.com/PrairieLearn/PrairieLearn/pull/12794/commits/d87e16f6ba4cab7b64cf73e4b566becb10d33851).